### PR TITLE
Add new connect splash screen content.

### LINKED ISSUE #8900: Activate Jetpack splash screen copy
When you activate Jetpack, we are showing you this modal full-screen:

image

I think the text is not explicit enough: it looks like connecting Jetpack will make it work better, or add extra capabilities: We are doing a pretty poor job communicating that the connection is required for Jetpack to even work. I think we should change the copy on that modal window to make this more clear.

@rickybanister ^

### PR DESCRIPTION
**Pull request by zinigor**
Fixes #8900 and #8617 

This still needs some work:
- [x] URLs for white buttons needed.
- [x] Scrolling doesn't work because of the absolutely positioned element.
- [x] The 'not now' button doesn't do anything.
- [x] Paddings and margins seem to be off.

I was hoping I could get some help with this from the design team.

screenshot from 2018-03-23 22-35-20

#### Changes proposed in this Pull Request:
* Adds new content, includes illustrations from #9097 
* Cleans up old unused styles.

#### Testing instructions:
* Deactivate Jetpack from the plugins menu.
* Activate Jetpack again, see the splash screen

#### Proposed changelog entry for your changes:
* Refreshed the connect splash screen to help new users learn more about Jetpack.

### CODE REVIEW

**Comment by joanrho** [comment]
> URLs for white buttons needed.

Chat with us: https://jetpack.com/contact-support/
Search our support site: https://jetpack.com/support/

**Comment by jeffgolenski** [comment]
Just some quick feedback!

[ ] There's like 6 different text styles on this one page, let's cut that back!
[ ] Jetpack Logo is called with an img tag — we can just inline that SVG to reduce a request!
[ ] Line-height of text in footer of the page needs to be taller. Line that starts with "Connect to, or create"
[ ] I'd like to call in @nicoleckohler for a quick review of the content here, just in case of any grammatical or flow issues


@joanrho The only goal of this page is to get people to connect. Do you think the "chat with us" or "search our support site" buttons are necessary?

**Comment by joanrho** [comment]
@jeffgolenski Thanks for the review and feedback! I'll get on the styling once we have the scrolling worked out.

As for the "chat with us" and "search our support" site buttons, that section already exists in the current longer disconnection full-screen banner, so my version updates that as a way for uncertain users to reach out for help before deciding. Here's what the current (old) one looks like:

01_wpadminjetpackdash_disconnected_old

I don't think we should nix this just yet, but we may decide to later on if we see that few users click through to those support options.

**Comment by zinigor** [comment]
Thanks for the review @joanrho and @jeffgolenski ! 

@jeffgolenski about this:
> Jetpack Logo is called with an img tag — we can just inline that SVG to reduce a request!

I'd argue here that one more request doesn't hurt that much, while we have the benefit to make changes to the logo in one place if needed. This PR also makes the settings masthead include that logo as a file.

**Comment by zinigor** [comment]
This is ready for review again, @jeffgolenski . Thanks for the feedback!

**Comment by rickybanister** [comment]
Did the font style issues get resolved? I know @zinigor is on a linux machine, but we're seeing both system fonts and helvetica in his screenshots.

File: _inc/client/components/masthead/index.jsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import analytics from 'lib/analytics';
  */
 import { getSiteConnectionStatus } from 'state/connection';
 import { getCurrentVersion, userCanEditPosts } from 'state/initial-state';
+import { imagePath } from 'constants/urls';
```
Comment (dereksmart) [inline _inc/client/components/masthead/index.jsx]: This is not defined when we're building the static.html file for the loading screen, so the console is always giving me a 404 for this path, even though the logo is showing after it loads.

**Comment by rickybanister** [comment]
Some things I noticed:

- [ ] The footer area is acting strange, the gray is partially covering the wp footer and the white overlay ends at a strange place 200px from the bottom:

image

- [ ] The typography all seemed small. On Jetpack.com the h1 and h2 are `2.5em` and compute to 35px. Here they are `1.35em` and compute to 22px. The terms of service text is `0.6875rem` and computes to 11px which is too small. Body copy on Jetpack.com's homepage is `1.25em` and here it's not set so it defaults to the `p` style which is 13px. I suppose we don't need to match jetpack.com exactly, but bumping up the headlines, paragraph styles, and small text would help. 

image

- [ ] This PR only covers the pop up splash content, not the disconnected dashboard (in the react codebase) stuff. We still need to take care of that.

**Comment by dereksmart** [comment]
I added a commit that renders the svg inline in the masthead, which fixes the undefined problem.  

It looks a little choppy to me though. 

screen shot 2018-04-24 at 5 47 02 pm

**Comment by rickybanister** [comment]
Let's merge this as is and address the choppy logo and other to-dos in another PR, is that alright @dereksmart?

@jeffgolenski can you take a look at the logo svg and the other checkboxed items I posted?

**Comment by joanrho** [comment]
>The footer area is acting strange, the gray is partially covering the wp footer and the white overlay ends at a strange place 200px from the bottom:

I'm not seeing this @rickybanister. Can you test again and see if you are?

**Comment by joanrho** [comment]
I've updated the styling so it better matches the hierarchies in my original mockups and removed the word "fascinating" from our TOS line in `functions.global.php`:

before after

Ready for review and to pass on to whoever for next steps (disconnected dashboard?).

cc @zinigor @dereksmart @rickybanister

**Comment by rickybanister** [comment]
That looks much better. 

Let's get this merged!

@zinigor @dereksmart, what's the best way to copy this to the dashboard, should we rewrite it and duplicate it? Are there any other viable solutions with less duplication? Can we use some interpretive thing for the links that get 'pre-processed' by the php and react code so we can reuse some meta template for the content and copy?

**Comment by dereksmart** [comment]
@rickybanister -- the good news is that @oskosk is currently working on a solution so we can reuse components in html/templates, so we only have the write them once. 

The bad new is that to take advantage of that, this whole PR would have to be re-written in React.

**Comment by keoshi** [comment]
@zinigor This seems to be in great shape to be closed—want to go ahead and merge it? Thanks!

**Pull request merged**